templates/todo/projects/python-mongo/.repo/bicep/repo.yaml

```diff
@@ -62,8 +62,13 @@ repo:
     - from: ../../../../../common
       to: ./
       ignore: 
-      - "infra/**/*"
-      - .devcontainer/**/*
+        - .github/**/*
+        - .devcontainer/**/*
+        - "infra/**/*"
+
+    # Github workflows for bicep
+    - from: .../../../../../common/.github/workflows/bicep
```

Comment:
path has extra ". " causing the file to not get copied on the template pr branch https://github.com/Azure-Samples/todo-python-mongo/compare/staging...pr/650

Reply:
Fixed.